packages/api-bindings/codegen-api.yml

```diff
@@ -41,6 +41,7 @@ config:
     ProfileId: ProfileId
     ProfileInterest: string
     ProxyActionId: string
+    PublicationUrl: Url
```

Comment:
The only reason I regenerated the GQL types was this resolving w/ `unknown` so I am avoiding several TS assertion to `string` with this.